### ISSUE TITLE
`hs.doc.builder` missing in 0.9.93?

### ISSUE THREAD

**Issue by dcreemer** [Hammerspoon/hammerspoon#3054]
From a console session:
```
> hs.doc.markdown
table: 0x600000bec940

> hs.doc.hsdocs
table: 0x600000ba94c0

> hs.doc.builder
...poon.app/Contents/Resources/extensions/hs/_coresetup.lua:662: module 'hs.doc.builder' not found:
	no field package.preload['hs.doc.builder']
...
```

**Comment by vladdoster** [comment]
Having this issue as well. You can use `0.9.91` as a stopgap until their is a new hotfix version released or figure out how to fix the issue.

**Comment by evantravers** [comment]
on `0.9.91`:

![image](https://user-images.githubusercontent.com/62209/148413759-bcfa2776-37a3-4d12-8b82-43fe1d50ced4.png)

**Comment by cmsj** [comment]
Apologies, this is fallout from reorganising the app bundle in 0.9.93. It'll be fixed in 0.9.94.
